packages/react-core/src/components/Menu/Menu.tsx

```diff
@@ -76,25 +73,7 @@ class MenuBase extends React.Component<MenuProps, MenuState> {
           {...getOUIAProps(Menu.displayName, ouiaId !== undefined ? ouiaId : this.state.ouiaStateId, ouiaSafe)}
           {...props}
         >
-          {hasSearchInput && (
-            <React.Fragment>
-              <div className={styles.menuSearch}>
-                <SearchInput
-                  value={this.state.searchInputValue}
-                  onChange={(value, event) => {
-                    this.setState({ searchInputValue: value });
-                    onSearchInputChange(event, value);
-                  }}
-                  onClear={event => {
-                    this.setState({ searchInputValue: '' });
-                    onSearchInputChange(event, '');
-                  }}
-                />
-              </div>
-              <Divider />
-            </React.Fragment>
-          )}
-          <div className={css(styles.menuContent)}>{children}</div>
+          <MenuContent>{children}</MenuContent>
```

Comment:
Doesn't the input have to be outside of the `MenuContent` ? If so then this should just be `{children}` and consumer will have to pass
```
<Menu>
  <MenuInput><TextInput /></MenuInput>
  <MenuContent>
     <MenuList>...</MenuList>
  </MenuContent>
</Menu>
```